modules/_labs/core/react/package.json

```diff
@@ -41,7 +41,9 @@
     "@workday/canvas-colors-web": "^0.17.13",
     "@workday/canvas-kit-react-core": "^3.2.0",
     "chroma-js": "^2.1.0",
-    "lodash": "^4.17.14"
+    "emotion-theming": "^10.0.10",
+    "lodash": "^4.17.14",
+    "rtl-css-js": "^1.13.1"
```

Comment:
Thanks for catching this 👍. Can you remove `emotion-theming` and `rtl-css-js` in the root package.json since we've lowered it to this module?

Reply:
Removing `emotion-theming` prevents stories from rendering